### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by tcrasset** [hajrashidimad/DL-wine-quality#1]
# Structured data review

## Good

- Good idea of doing one hot encoding and some kind of normalization --> could have been in a separate file too
- The training is successful, the loss is decreasing

## To improve

- What is your reasoning for doing `(x-x.min)/(x.ax - x.min) * 10` ? Why `*10` and why only for a few features
- What is the conclusion ? What steps improved the score ?
- What is the baseline ?
- You should have used a validation set
